packages/modules/atlas-web-content/scripts/build.ts

```diff
@@ -40,6 +43,47 @@ async function main(): Promise<void> {
 
         await copyStylesAndAssets(mode === "start", outputDir);
     }
+
+    await buildAndCopyWidgets(
+        mode,
+        ["badge-web", "maps-web", "progress-bar-web", "progress-circle-web", "timeline-web"],
+        outputDir
+    );
+}
+
+async function buildAndCopyWidgets(command = "build", widgets: string[] = [], destination?: string) {
```

Comment:
Can we not somehow share this code between the module scripts? 😄

Reply:
They live in different packages, so it should be extracted to some module instead.. dont know if it worth

Reply:
ok!